dor-workflow-service.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_dependency 'activesupport', '>= 3.2.1', '< 5'
-  gem.add_dependency 'nokogiri', '~> 1.6.0'
+  gem.add_dependency 'nokogiri', '~> 1.7'
```

Comment:
Great catch, that is why the builds are failing on some of these PRs, because our build matrix includes ruby < 2.1.   See also https://github.com/sul-dlss/dor-workflow-service/pull/41